CONTRIBUTING.md

```diff
@@ -83,19 +76,29 @@ if you choose to install `nextest`.
 
 ### Development
 
-After cloning the repository, run Ruff locally from the repository root with:
+After cloning the repository, Ruff can be run locally from the repository root on a file such as `add_rule.py` using:
 
 ```shell
-cargo run -p ruff -- check /path/to/file.py --no-cache
+cargo run -p ruff -- check ./scripts/add_rule.py --no-cache
 ```
 
-Prior to opening a pull request, ensure that your code has been auto-formatted,
-and that it passes both the lint and test validation checks:
+Now that the repository is cloned, you can optionally install pre-commit hooks to automatically run the validation checks
+when making a commit:
 
 ```shell
-cargo clippy --workspace --all-targets --all-features -- -D warnings  # Rust linting
-RUFF_UPDATE_SCHEMA=1 cargo test  # Rust testing and updating ruff.schema.json
-pre-commit run --all-files --show-diff-on-failure  # Rust and Python formatting, Markdown and Python linting, etc.
```

Comment:
code blocks give the user an ability to copy and paste into a terminal, but doing this doesn't work since the trailing comments break the newlines.